### ISSUE TITLE
My project stuck in "Running calculateMetadata()" step after updating Remotion

### ISSUE THREAD

**Issue by mnlbox** [remotion-dev/remotion#3584]
# Bug Report 🐛
Hi @JonnyBurger,

I recently updated all of my Remotion packages to the latest version `4.0.125` and now when I run my project with `studio` command it stuck in "Running calculateMetadata()" step like this:
<img width="892" alt="image" src="https://github.com/remotion-dev/remotion/assets/25859075/601b57bd-3ebd-4826-baf3-ceff493e8fae">

I also checked my console. There is no error or warning there.
In a meantime I can downgrade to an older version but I just wanted to report it here as well.

**Comment by mnlbox** [comment]
Update:
I downgraded from `4.0.125` to **4.0.122** as it seems my project is not working with `4.0.123` and `4.0.124` as well.

**Comment by mnlbox** [comment]
Update:
I tried with the the newest version `4.0.126` and it also have same issue. So as previous, the latest working version for me is `4.0.122`

**Comment by JonnyBurger** [comment]
I tried this in our testbed and cannot see that an arbitrary calculateMetadata() function is not resolving. 

Is there something in the console?
Any async function that may not be resolved? Can you narrow down with console.log's where it gets stuck?
Do you want to post your calculateMetadata() function?

**Comment by mnlbox** [comment]
Hi @JonnyBurger  Thanks for your reply,

I removed my node_modules and PNPM cache and then reinstalled my packages again and now it's working even with the latest version `4.0.126` I don't know if it's a problem only with PNPM or not but now all good and there is no issue for running Remotion studio. 

Currently, I just have a problem with rendering my project that I reported that in another issue. I'm closing this issue for now and will follow-up the rendering issue in other issue ([#3587](https://github.com/remotion-dev/remotion/issues/3587)).
